pages/common/exec.md

```diff
@@ -6,14 +6,14 @@
 
 `exec {{command -with -flags}}`
 
-- Replace with the specified command using initialized environment variables:
+- Replace with the specified command clearing environment variables:
```

Comment:
Should there be a comma after "command" ?

Reply:
Hmm I was kind of unsure, but ok, added.